stubs/channels/channels/generic/websocket.pyi

```diff
@@ -0,0 +1,65 @@
+from typing import Any
+
+from asgiref.typing import WebSocketConnectEvent, WebSocketDisconnectEvent, WebSocketReceiveEvent
+from channels.consumer import AsyncConsumer, SyncConsumer, _ChannelScope
+from channels.layers import BaseChannelLayer
+
+class WebsocketConsumer(SyncConsumer):
+    groups: list[str] | None
+    scope: _ChannelScope
+    channel_name: str
+    channel_layer: BaseChannelLayer
+    channel_receive: Any
+    base_send: Any
```

Comment:
Considering these are inherited from `AsyncConsumer`, they shouldn't be repeated here.


```suggestion
```

I think the same is actually true for some of the methods below, but I didn't check (and that is non-critical).